_layouts/base.html

```diff
@@ -11,7 +11,7 @@
 {% include youtube-embed-head.html id=youtube_id %}
 {% endif %}
     <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400i,700,700i|Signika:600,700" rel="stylesheet">
-    <link rel="stylesheet" href="/static/css/styles.css">
+    <link rel="stylesheet" href="/static/main.css">
```

Comment:
@mtrythall this path change is what broke the css outside of the homepage. The homepage only uses inline styles from what I'm seeing. 

I'm updating my local setup to test and then I'll change it back unless you updated this for some other reason.

Reply:
@mtrythall on second thought, I think we are seeing caching.